sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -59,14 +60,14 @@ export interface SearchStudentRespDto {
 }
 
 /**
- * DTO Object for student restriction.
+ * DTO for student restriction.
  * This object is returned by controller.
  */
-export interface StudentRestrictionDTO {
-  hasRestriction: boolean;
-  hasFederalRestriction: boolean;
-  hasProvincialRestriction: boolean;
-  restrictionMessage: string;
+export class StudentRestrictionAPIOutDTO {
+  //  code, is the restriction code.
```

Comment:
Please add the proper comments. In this way VS Code will not be able to display them, right?